src/plugins/pcr/src/InSilicoPcrWorker.cpp

```diff
@@ -410,11 +410,15 @@ QByteArray InSilicoPcrReportTask::productsTable() const {
     return chapterName(tr("Products count table")) + chapterContent(result);
 }
 
-QByteArray InSilicoPcrReportTask::primerDetails() const {
+QByteArray InSilicoPcrReportTask::primerDetails() {
     QByteArray result;
     for (int i = 0; i < primers.size(); i++) {
         QPair<Primer, Primer> pair = primers[i];
         PrimersPairStatistics calc(pair.first.sequence.toLocal8Bit(), pair.second.sequence.toLocal8Bit());
+        if (!calc.getInitializationError().isEmpty()) {
+            setError(calc.getInitializationError());
```

Comment:
You added the error, but there is no check of `stateInfo` where you get this function called